src/Tracing/Instrumentation/Messenger/StrategyStamp.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the worldia/instrumentation-bundle package.
+ * (c) Worldia <developers@worldia.com>
+ */
+
+namespace Instrumentation\Tracing\Instrumentation\Messenger;
+
+use Symfony\Component\Messenger\Stamp\StampInterface;
+
+final class StrategyStamp implements StampInterface
```

Comment:
I would suggest to rename this stamp into something less generic, maybe `LinkStrategyStamp` ?
Or to move it into the propagation namespace, but then it's kind of weird to have the stamp in another process than the one handling it.

Reply:
👍